back/src/domains/inclusion-connected-users/use-cases/GetInclusionConnectedUser.unit.test.ts

```diff
@@ -186,303 +186,306 @@ describe("GetUserAgencyDashboardUrl", () => {
-
-        const fakeBusinessContact = new ContactEntityBuilder()
-          .withEmail(john.email)
-          .build();
-
-        const establishmentAggregate1 = new EstablishmentAggregateBuilder()
-          .withEstablishmentSiret("89114285300012")
-          .withContact(fakeBusinessContact)
-          .build();
-
-        const establishmentAggregate2 = new EstablishmentAggregateBuilder()
-          .withEstablishmentSiret("89114285300013")
-          .withContact(fakeBusinessContact)
-          .build();
 
-        uow.establishmentAggregateRepository.establishmentAggregates = [
-          establishmentAggregate1,
-          establishmentAggregate2,
-        ];
-
-        const result = await getInclusionConnectedUser.execute(
-          undefined,
-          inclusionConnectJwtPayload,
-        );
+      describe("establishments", () => {
+        it("retrieve establishments when IC user is establishement rep in at least one establishment", async () => {
+          uow.userRepository.setInclusionConnectedUsers([
+            {
+              ...randomUser,
+              dashboards: { agencies: {}, establishments: {} },
+              agencyRights: [],
+            },
+          ]);
+
+          const fakeBusinessContact = new ContactEntityBuilder()
+            .withEmail(randomUser.email)
+            .build();
+
+          const establishmentAggregate1 = new EstablishmentAggregateBuilder()
+            .withEstablishmentSiret("89114285300012")
+            .withContact(fakeBusinessContact)
+            .build();
+
+          const establishmentAggregate2 = new EstablishmentAggregateBuilder()
+            .withEstablishmentSiret("89114285300013")
+            .withContact(fakeBusinessContact)
+            .build();
+
+          uow.establishmentAggregateRepository.establishmentAggregates = [
+            establishmentAggregate1,
+            establishmentAggregate2,
+          ];
+
+          const result = await getInclusionConnectedUser.execute(
+            {},
+            randomUser,
+          );
+
+          expectToEqual(result.establishments, [
+            {
+              siret: establishmentAggregate1.establishment.siret,
+              businessName: establishmentAggregate1.establishment.name,
+            },
+            {
+              siret: establishmentAggregate2.establishment.siret,
+              businessName: establishmentAggregate2.establishment.name,
+            },
+          ]);
+        });
 
-        expectToEqual(result.establishments, [
-          {
-            siret: establishmentAggregate1.establishment.siret,
-            businessName: establishmentAggregate1.establishment.name,
-          },
-          {
-            siret: establishmentAggregate2.establishment.siret,
-            businessName: establishmentAggregate2.establishment.name,
-          },
-        ]);
+        it("do not retrieve  establishment  when IC user is not establishment representative in at least one establishment", async () => {
```

Comment:
espaces en trop dans cette phrase ^^

```suggestion
        it("do not retrieve establishment when IC user is not establishment representative in at least one establishment", async () => {
```